test/test_short_name_claims.js

```diff
@@ -60,126 +70,188 @@ contract('ShortNameClaims', function (accounts) {
 
 		const priceOracle = await SimplePriceOracle.new(1);
 
-		claims = await ShortNameClaims.new(priceOracle.address, registrar.address);
+		claims = await ShortNameClaims.new(priceOracle.address, registrar.address, ratifierAccount);
 		await registrar.addController(claims.address, {from: ownerAccount});
 		await registrar.transferOwnership(registrarOwner);
 	});
 
 	it('should permit a DNS name owner to register a claim on an exact match', async () => {
-		const tx = await claims.submitExactClaim(dns.hexEncodeName('foo.test.'), claimantAccount, {value: 31536001});
+		const tx = await claims.submitExactClaim(dns.hexEncodeName('foo.test.'), claimantAccount, 'test@example.com', {value: 31536001});
```

Comment:
I wonder how many people hesitate to put their email address on blockchain (even on event) as plain text (unless they use throw away email). Are we not offering ways for people to put encrypted data which only we can decrypt?

Reply:
We need some way to contact users. We could add encryption, but it's an extra layer of complexity, and users can easily create throwaway addresses. It'll be clearly stated in the UI that this will be published publicly.